KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift

```diff
@@ -14,6 +14,10 @@ import Combine
 
 final class KarhooBookingViewController: UIViewController, BookingView {
 
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        .lightContent
```

Comment:
What if a demand partner chooses a lighter primary color to override ours? Maybe we could adapt this style depending on the brightness of the primary color. There is already a property that figures this out in the UIColor extension, it's called `isLight`.